javascript/src/test/components/BasicCourseSearch/BasicCourseTable.test.js

```diff
@@ -191,6 +196,44 @@ describe("BasicCourseTable tests", () => {
     expect( queryAllByText("TBD").length).toBe(5);
   });
 
+  // Testing styling for classes w/more than one section
+  test("check that full sections have a red background", () => {
+  	const {getByText} = render(<BasicCourseTable classes = {courseFixtures.sectionColor} />);
+  	expect( getBackgroundColor(getByText, "07500") ).toBe("#FF0000");
+  });
+
+  test("check that almost full sections have a orange background", () => {
+  	const {getByText} = render(<BasicCourseTable classes = {courseFixtures.sectionColor} />);
+  	expect( getBackgroundColor(getByText, "07501") ).toBe("#FFBF00");
+  });
+
+  test("check available sections have a light blue", () => {
+  	const {getByText} = render(<BasicCourseTable classes = {courseFixtures.sectionColor} />);
+  	expect( getBackgroundColor(getByText, "07502") ).toBe("#EDF3FE");
+  });
+
+  test("check that class associated with sections is dark blue", () => {
```

Comment:
This lecture with enroll code 07492 is full in your test data, I thought this means it should be red? Why is it dark blue. I think I am confused by this case if you can explain. 
//If it is not a stand alone lecture that is unavailable or full and is just a class set it to dark blue.

What is difference between a class and lecture